proto/intertx/query.proto

```diff
@@ -5,11 +5,14 @@ package intertx;
 option go_package = "github.com/cosmos/interchain-accounts/x/inter-tx/types";
 
 import "gogoproto/gogo.proto";
+import "google/api/annotations.proto";
 
 // Query defines the gRPC querier service.
 service Query {
   // QueryInterchainAccountFromAddress returns the interchain account for given owner address on a given connection pair
-  rpc InterchainAccountFromAddress(QueryInterchainAccountFromAddressRequest) returns (QueryInterchainAccountFromAddressResponse);
+  rpc InterchainAccountFromAddress(QueryInterchainAccountFromAddressRequest) returns (QueryInterchainAccountFromAddressResponse) {
+    option (google.api.http).get = "/inter-tx/v1beta1/interchain_account_from_address/{owner}/{connection_id}";
```

Comment:
Thoughts on removing `FromAddress` from this? 
We could do it in another PR.

Reply:
I think for `vtbeta1` the `query.proto` file should be in a directory `proto/intertx/v1beta1/query.proto` and proto package should be updated.

Otherwise you could omit the `v1beta1` in the URL.